CONTRIBUTING.md

```diff
@@ -22,38 +22,58 @@ To run the test suite, call `./test.sh` from your local machine - the script wil
 
 ## Testing
 
-### Running Tests
+### Running Unit Tests
 
-To run the service broker unit tests run:
+To run the Conjur Service Broker unit tests, run the following:
 ```sh-session
 $ # Build the base images and artifacts
 $ ./build.sh
 
-$ # Run the tests
-$ ./test.sh
+$ # Run the unit tests
+$ ./bin/test_unit
 ```
 
-_Note: Tests rely on having built `conjur-service-broker` and `conjur-service-broker-test`
+### Running Local Integration Tests
+
+The [bin/test_integration](./bin/test_integration) script provides a full suite of integration tests
+for testing Service Broker functionality against Conjur. The tests use Docker
+Compose to spin up local instances of Conjur and Service Brokers, so the
+tests can be run locally.
+
+To run the Service Broker local integration tests, run the following:
+```sh-session
+$ # Build the base images and artifacts
+$ ./build.sh
+
+$ # Run the integration tests
+$ ./bin/test_integration
+```
+
+_Note: The integration tests rely on having built `conjur-service-broker` and `conjur-service-broker-test`
 images available. If you make changes to your local repository and would like to see those
 changes reflected in the test containers, either re-run `./build.sh` or run
 `docker-compose build <service_name>` to rebuild the source image(s) before running the
 tests._
 
-### Integration Testing
+### End-to-End (E2E) Integration Testing
 
-The Conjur Service Broker integration tests have external dependencies to run successfully:
+The Conjur Service Broker End-to-End integration tests have external dependencies to run successfully:
 
 * A Cloud Foundry foundation (version 2.4)
 * A Conjur instance accessible by the test runner and by the Cloud Foundry instance above
     > The configuration and policy for this conjur instance are defined in `./ci/integration/conjur`
 
 The connection information and credentials for these service are provided by Summon to the test runner.
 
-See [secrets.yml](./secrets.yml) for the variables required to run the tests.
+See [ci/secrets.yml](./ci/secrets.yml) for the variables required to run the tests.
 
-Once Summon is configured when the connection information, the integration tests may be executed byt running:
+Once Summon is configured when the connection information, the integration tests may be executed by running:
 ```sh-session
-$ summon ./test.sh
+$ # Build the base images and artifacts
+$ ./build.sh
+
+$ cd ci
+$ summon ./test_e2e
```

Comment:
Suggestion:
To run the end-to-end tests, first build the base images and artifacts:
```shell script 
./build.sh
```
Then, `cd` into the `ci` directory, and execute the test-runner with `summon`:
```shell script
cd ci
summon ./test_e2e
```